bundles/persistence/org.openhab.persistence.mysql/java/org/openhab/persistence/mysql/internal/MysqlPersistenceService.java

```diff
@@ -153,6 +157,35 @@ public void unsetPersistentStateRestorer(PersistentStateRestorer persistentState
 	public String getName() {
 		return "mysql";
 	}
+	
+	/**
+	 * 
+	 * @param i
+	 * @return
+	 */
+	private String getItemType(Item i) {
+		Item item = i;		
+		if(i instanceof GroupItem){
+			logger.debug("mySQL: Item is GroupItem, try to find ItemType by BaseItem");
```

Comment:
please add the itemName to log statement (using parameterised logging) for the sake of readability of the logs (also applies to the subsequent statements) … image you only have the log at hand and try to give advice where the problem could be.